modules/log6x-6-1-0-rn.adoc

```diff
@@ -0,0 +1,53 @@
+= Logging 6.1.0 Release Notes
+// Module included in the following assemblies:
+//
+
+:mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-1-0_{context}"]
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:9038[{OCP-short}{logging} Release 6.1.0].
+
+[id="openshift-logging-release-notes-6-1-0-enhancements"]
+== New Features and Enhancements
+
+=== Log Collection
+
+* This enhancement adds the source `iostream` to the attributes sent from collected container logs. The value is set to either `stdout` or `stderr` based on how the collector received it. (link:https://issues.redhat.com/browse/LOG-5292[LOG-5292])
+
+* With this update, the default memory limit for the collector increases from 1024Mi to 2024Mi. Users should adjust resource limits based on their cluster’s specific needs and specifications. (link:https://issues.redhat.com/browse/LOG-6072[LOG-6072])
+
+* With this update, users can now set the syslog output delivery mode to either `AtLeastOnce` or `AtMostOnce.` (link:https://issues.redhat.com/browse/LOG-6355[LOG-6355])
+
+=== Log Storage
+
+* With this update, the new `1x.pico` LokiStack size supports clusters with fewer workloads and lower log volumes (up to 50GB/day). (link:https://issues.redhat.com/browse/LOG-5939[LOG-5939])
+
+[id="logging-release-notes-6-1-0-technology-preview-features"]
+== Technology Preview
+
+With this update, the `ClusterLogForwarder` resource now supports forwarding logs to OpenTelemetry-compatible receivers using HTTP/JSON, as defined by the OpenTelemetry OTLP specification. This enhancement allows logs to be transported to Red Hat Managed Lokistack or other OTLP-compliant services.
```

Comment:
This specific release is only related to sending to lokiStack, although the "tech-preview" relates to anything OTLP at this point.   We previously released the otlp (generic) output in 6.0, which sends to compatible receivers.   For 6.1, we are adding the feature to the lokiStack output spec, via the new `dataModel` field.

Reply:
Additionally, or maybe not worth the mention, but our lokiStack now "accepts" the collector's otel logs, which is the other part of the feature (along with query and UI work).

Reply:
This will be covered in https://issues.redhat.com/browse/OBSDOCS-1494